src/core/amount.js

```diff
@@ -105,6 +105,19 @@ Amount.NaN = function() {
   return result;                      // but let's be careful
 };
 
+Amount.createFast = function(value: Value, currency: Currency, issuer: UInt160,
```

Comment:
to make the name more consistent with other names, maybe we could call it `from_components`?